docker-compose/high-availability/docker-compose.yaml

```diff
@@ -2,7 +2,7 @@ version: '3.0'
 
 services:
   db:
-    image: timescaledev/promscale-extension:latest-pg12
+    image: timescaledev/promscale-extension:latest-ts2-pg14
```

Comment:
lets actually change this to `timescale/timescaledb-ha:pg14-latest`.

Reply:
I think we need to produce an HA version of the extension image and use that.

Reply:
@cevian Do you know why we were not using timescaledb-ha previously?

Reply:
@bitwalker the extension should already be in the HA image

Reply:
@JamesGuthrie I am guessing legacy oversight

Reply:
done